doc/vim-go.txt

```diff
@@ -1762,11 +1765,9 @@ binaries).
 
 Vim becomes slow while editing Go files~
 
-Don't enable these options:
->
-  let g:go_highlight_types = 0
-  let g:go_highlight_operators = 0
-<
+Don't enable the |'g:go_highlight_types'| and |'g:go_highlight_operators'|
```

Comment:
While we're at it, can the language be changed to a positive assertion?
> Disable the ...

Please include `g:go_highlight_assignments` in this list.

Reply:
I think there are actually a bunch of other things that can cause performance problems too; I've updated the entry.